### PR TITLE
chore (docs): add a note about security requirements on macOS

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

File: docs/modules/ROOT/pages/cli/cli.adoc [hunk 1]
```diff
@@ -8,6 +8,16 @@ Releases of the Camel K CLI are available on:
...
 - Homebrew (Mac and Linux): https://formulae.brew.sh/formula/kamel
 
+== Running on macOS
+
+Before running the CLI on macOS, it may be necessary to give adequate permissions for it to run.
+
+It can either be done on the "Privacy & Security" panel in the System Settings or via command-line:
+
+```
+xattr -dr com.apple.quarantine ~/path/to/kamel
```
Comment (essobedo) [inline docs/modules/ROOT/pages/cli/cli.adoc:18]: I'm just wondering if we need the `r` flag?

**Review by essobedo — APPROVED** [review]

File: docs/modules/ROOT/pages/cli/cli.adoc (hunk 1 above)
Comment (essobedo) [inline docs/modules/ROOT/pages/cli/cli.adoc:18]: Maybe we could simply keep `/path/to/kamel` as path to prevent potential confusion

Comment (orpiske) [reply docs/modules/ROOT/pages/cli/cli.adoc:18]: +1

Comment (orpiske) [reply docs/modules/ROOT/pages/cli/cli.adoc:18]: I've just tested here and we don't need that. 👍

**Pull request merged**